src/components/Boostagram.js

```diff
@@ -37,11 +58,29 @@ const skipTo = (seconds) => {
 }
 
 const Boostagram = ({ boostagram }) => {
-  const boostEmojis = ['🚀', '⚡️', '💬', '📣', '🧡']
-  const boostEmoji = boostEmojis[Math.floor(Math.random() * boostEmojis.length)]
+  const apps = {
+    Breez: {
+      icon: <AppIcon content="/assets/podcast/icon_breez.png" />,
```

Comment:
`content` does not seem to work in Firefox (see [preview url](https://deploy-preview-53--seetee.netlify.app/podcast/s2e1/13-gigi-the-future-of-value/)).

Tested using `src` - works in both, chrome and firefox.
```
<AppIcon src="/assets/podcast/icon_breez.png" />
```

Reply:
Strange. It seems to works for me in Firefox (see compatibility list [here](https://developer.mozilla.org/en-US/docs/Web/CSS/content#browser_compatibility)). What version are you running?

Reply:
I can just switch it to `src` though, should be fine as well.

Reply:
Tested with Firefox v95. I have not seen `content` without using `:before` or `:after` pseudo-elements before, so I might not be up-to-date with its state of the art usage.

Reply:
Yeah maybe that was it. Strange that it worked in my Firefox (v96). Nevertheless, `src` is more straight forward anyway so 👍 .